pages/common/code.md

```diff
@@ -0,0 +1,15 @@
+# code
+
+> Visual Studio Code.
+
+- Open VS Code:
+
+`code`
+
+- Open current directory in VS Code:
```

Comment:
Perhaps `Open the current directory in VS Code:` would read better?

Reply:
Oh yeah. I've almost stopped using English articles ever since I started to make my commit messages shorter 😅